README.md

```diff
@@ -91,8 +98,24 @@ DATABASES['default']['NAME'] = 'var/workbench.db'
 ```
 
 Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
+before starting the workbench. `problem-builder` uses MySQL 5.6 and hence you can quickly spin up
+an instance of MySQL 5.6 with docker. Run the following command to start a MySQL 5.6 server.
+
+```bash
+docker run --rm -it -p 3306:3306 -e MYSQL_ROOT_PASSWORD=rootpw -e MYSQL_DATABASE=db circleci/mysql:5.6 mysqld --character-set-server=latin1 --collation-server=latin1_swedish_ci
+```
+
+By default, the `xblock-sdk` uses the SQLite database by default but MySQL
```

Comment:
By default is coming twice here. If we mention the reliance on SQLite before, we can just say that the database config can be overridden using environment variables. 

That said, the readme explicitly has you define the database in line 97, so this setting will just not be used. You need to remove that line as well.

Reply:
I have updated the paragraph.